src/signals/core.nim

```diff
@@ -49,7 +49,7 @@ QtObject:
     let signalString = $jsonSignal["type"].getStr
 
     trace "Raw signal data", data = $jsonSignal
-
+    
```

Comment:
🤔

Reply:
I want a linter just to avoid that :dagger:

Reply:
Have you tried `nimpretty`?